src/components/dropdown.jsx

```diff
@@ -22,14 +21,6 @@ class Dropdown extends React.Component {
     mediaQuery.addEventListener('change', this.updateMediaQuery);
```

Comment:
This seems to be missing the corresponding `mediaQuery.removeEventListener` in `componentWillUnmount`.